applications/crossbar/src/api_resource.erl

```diff
@@ -1071,28 +1084,33 @@ process_chunk(#{context := Context
                }=ChunkMap) ->
     case cb_context:resp_data(Context) of
         0 ->
+            lager:debug("resp data is 0, resetting"),
```

Comment:
Can you say something like
```erlang
lager:debug("(chunked) ~s did not send any resp data", [_ToFun]),
```

Same for log lines below too, add `(chunked)` and how many row/data was sent instead.